### PR TITLE
style: set up linting and commit hook

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,12 +14,12 @@
     "start": "webpack-dev-server --mode development --config webpack.config.cjs"
   },
   "dependencies": {
+    "@graphy/content.ttl.read": "^4.3.4",
```
Comment (cristianvasquez) [inline package.json:17]: Nothing against it, but what provides @graphy/content.ttl that @rdf-esm/formats-common doesn't?

**Review by cristianvasquez — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (tpluscode) [reply package.json:17]: A parser is only used to load the sample resources. The formats package uses n3, which acts weird when bundled in wc.dev. Strangely, it appears as a `this` inside rdf-ext's `blankNode` which throws because there is an undefined property.

tl;dr; must be caused by the fact that there are duplicates of some of those packages where newer are ESM and older are CJS
